homeassistant/components/zwave_js/strings.json

```
@@ -40,6 +40,7 @@
     "step": {
       "configure_addon_user": {
         "data": {
+          "socket_path": "Socket device path",
```

We can also just use "Socket path" if that's better:
```suggestion
          "socket_path": "Socket path",
```
I've used "Socket device path", as the `usb_path` is translated with "USB device path".
In ZHA, we just use "Device path" (for allowing both USB and socket entries when entering the path manually).